guides/common/modules/proc_configuring-the-active-directory-authentication-source-on-projectserver.adoc

```diff
@@ -105,6 +105,31 @@ $ curl -k -u : --negotiate https://{foreman-example-com}/users/extlogin
 <html><body>You are being <a href="{foreman-example-com}/hosts">redirected</a>.</body></html>
 ----
 
+.Troubleshooting
+* Connecting to the AD LDAP can sometimes fail with an error such as the following:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+{ "error": { "message": "ERF77-7629 [Foreman::LdapException]: Error while connecting to 'server.com' LDAP server at 'ldap.example.com' during authentication ([Net::LDAP::Error]: Connection reset by peer - SSL_connect)" } }
```

Comment:
Technically this is API output. I think the `message` part can also be seen in the UI. Probably if you use the `Test connection` button if it's 100% reproducible but I don't have the setup to test this.

Reply:
What if I clarify that the error message appears in the logs? That's how the issue was troubleshooted. Does it make sense?

Reply:
> if it's 100% reproducible

Sadly it's not, the failure is intermittent. From what I've seen I'd say each ldap lookup has a 1:50 chance of hitting this.

> What if I clarify that the error message appears in the logs?

That would make sense to me